src/analysis/inner_product_space/positive.lean

```diff
@@ -58,6 +58,18 @@ and `∀ x : V, 0 ≤ re ⟪T x, x⟫` -/
 def is_positive (T : E →ₗ[𝕜] E) : Prop :=
 T.is_symmetric ∧ ∀ x : E, 0 ≤ re ⟪T x, x⟫
 
+lemma is_positive.is_symmetric {T : E →ₗ[𝕜] E} (hT : is_positive T) :
```

Comment:
```suggestion
protected lemma is_positive.is_symmetric {T : E →ₗ[𝕜] E} (hT : is_positive T) :
```

Reply:
what does `protected` do?

Reply:
It prevents you being able to access the declaration by `open`ing the relevant namespace. In that case, it avoids Lean being confused between `linear_map.is_symmetric` and `linear_map.is_positive.is_symmetric` if ever you open `is_positive` inside the `linear_map` namespace.